response_operations_ui/views/info.py

```diff
@@ -1,31 +1,28 @@
-from json import JSONDecodeError, loads
+import json
 import logging
 from pathlib import Path
 
-from flask import make_response, jsonify
+from flask import Blueprint, make_response, jsonify
 from structlog import wrap_logger
 
-from response_operations_ui import app
-
 
 logger = wrap_logger(logging.getLogger(__name__))
 
+info_bp = Blueprint('info_bp', __name__, static_folder='static', template_folder='templates')
 
-@app.route('/info', methods=['GET'])
-def get_info():
 
-    _health_check = {}
-    if Path('git_info').exists():
-        with open('git_info') as io:
-            try:
-                _health_check = loads(io.read())
-            except JSONDecodeError as e:
-                logger.error('Failed to decode git_info json', exc_info=e)
+_health_check = {}
+if Path('git_info').exists():
+    with open('git_info') as io:
+        _health_check = json.loads(io.read())
 
+
+@info_bp.route('/', methods=['GET'])
+def get_info():
     info = {
-        "name": 'response-operations-ui',
-        "version": '0.0.1',
+        "name": "response-operations-ui",
+        "version": "0.0.1",
     }
-    info = {**_health_check, **info}
+    info = dict(_health_check, **info)
```

Comment:
This all works fine but just out of interest is there an advantage to this syntax over the previous one?

Reply:
More for readability, shouldn't be any difference